### PR TITLE
Update quickstart.rst to include example of passing server arguments

### PR DESCRIPTION
**Pull request by rkrishnasanka**
Changes proposed in this pull request:
 - Updated the docs such that it comes with an example for passing server arguments and serving static files

### CODE REVIEW

File: docs/quickstart.rst [hunk 1]
```diff
@@ -96,6 +96,15 @@ to ``tornado`` or ``gevent``:
     app = connexion.FlaskApp(__name__, port = 8080, specification_dir='openapi/', server='tornado')
 
 
+Additoinally you can pass the parameters for the corresponding server as a dictionary using the `server_args` parameter. Hence in scenarios like serving static content, you can pass the parameters as given here:
```
Comment (Ruwann) [inline docs/quickstart.rst:99]:
```suggestion
Additionally, you can pass the parameters for the corresponding server as a dictionary using the `server_args` parameter. For example, in a scenario where you're serving static content, you can pass the parameters as follows:
```

File: docs/quickstart.rst [hunk 2]
```diff
@@ -96,6 +96,15 @@ to ``tornado`` or ``gevent``:
...
+
+.. code-block:: python
+
+    import connexion
+
+    app = connexion.FlaskApp(__name__, port = 8080, specification_dir='openapi/', server='flask', server_args={'static_url_path':"/", 'static_folder':"whereever/your/static/files/are"})
```
Comment (Ruwann) [inline docs/quickstart.rst:105]:
```suggestion
    app = connexion.FlaskApp(
        __name__,
        port=8080,
        specification_dir='openapi/',
        server='flask',
        server_args={'static_url_path': '/', 'static_folder': 'wherever/your/static/files/are'}
    )
```

**Review by Ruwann — APPROVED** [review]
Thanks for the PR! 
I have made some small adjustments.
Full disclaimer: this will probably be changed when updating the docs to 3.0 :)

**Comment by rkrishnasanka** [comment]
No issues !

It took me like 24 hours to figure out how to pass the arguments, so I think it's important to highlight this in the documentation.

**Pull request merged**